mlflow/store/dbmodels/models.py

```diff
@@ -407,3 +407,27 @@ def to_mlflow_entity(self):
         return Param(
             key=self.key,
             value=self.value)
+
+
+class SqlPeriodicJobs(Base):
+    """
+    DB model for periodic jobs. These are recorded in ``periodic_jobs`` table.
```

Comment:
neatpicking: use single `.
```suggestion
    DB model for periodic jobs. These are recorded in `periodic_jobs` table.
```